Sources/SwiftInspectorVisitors/Tests/PropertyVisitorSpec.swift

```diff
@@ -373,7 +373,7 @@ final class PropertyVisitorSpec: QuickSpec {
               name: "foo",
               typeDescription: nil,
               modifiers: [.internal, .instance],
-              paradigm: .computedVariable("{ Foo() }"))
+              paradigm: .computedVariable("Foo()"))
```

Comment:
This is nice symmetry, since the `computedVariable` case indicates that we will have `{` and `}`.